src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -127,11 +147,58 @@ public string Url
             set { SetObject(SchemaRegistryConfig.PropertyNames.SchemaRegistryRequestTimeoutMs, value.ToString()); }
         }
 
+        /// <summary>
+        ///     File or directory path to CA certificate(s) for verifying the schema registry's key.
+        ///
+        ///     default: ''
+        ///     importance: low
+        /// </summary>
+        public string SslCaLocation
+        {
+            get { return Get(SchemaRegistryConfig.PropertyNames.SslCaLocation); }
+            set { SetObject(SchemaRegistryConfig.PropertyNames.SslCaLocation, value.ToString()); }
+        }
+
+        /// <summary>
+        ///     Path to client's keystore (PKCS#12) used for authentication.
```

Comment:
ah, here it is! 

I don't really like the keystore naming since that is a Java thing.

Reply:
Hi @edenhill, 

I agree, but after checking ProducerConfig or ConsumerConfig configs and to keep the same convention I changed to SslKeystoreLocation, both are PKCS#12 format

Reply:
I think that since a precedent has been set, we should go with the keystore nomenclature, even though i don't like it either. Bikeshedding a bit.

Reply:
I think it was a mistake to call them Keystore (a specific format which is not PKCS12) in the first place, since they are not.
I would rather see us use a proper name for these new properties, and adding aliases in the kafka client with proper names and deprecating the keystore, rather than dig deeper down this hole.
Calling it keystore will not be helpful for our users since they think they can pass a JKS, with all kinds of troubles ahead of them.

Reply:
Matt reminded me that librdkafka has a ssl.keystore.location which takes a PKCS#12, very much like Certificate2, so I redact all my previous opinions and happily cheer on SslKeyStoreLocation, et.al.